certified-connectors/DocTOPDFConnector/Readme.md

```diff
@@ -0,0 +1,13 @@
+This custom connector is used to convert MS Word document to PDF file with fidelity in imbedded charts, graphs and graphics. This connector calls our company API service which passes the document and credentials , the API service receives the MS Word document and all content and convert it PDF file., once the conversion is completed the API send it back as a PDF to the connector / environment, depending on the configuration of the workflow the file is then deposited in the identified location. We created this tool for our clients to serve a need and feel this may be a connector that has a space in the market
```

Comment:
@yispot You are submitting your connector as "certified-connector" but mentioned as "custom-connector". This might be incorrect placement or the description of your connector needs be fixed.

Reply:
changed the name.